gallery-so/gallery
# [GAL-4018] remove caption step from save gallery

### Summary of Changes

Remove Add Caption step from edit gallery flow, now that everyone has access to Posts.

### Demo or Before/After Pics
now after editing a gallery, the user clicks SAVE, then DONE. 


https://github.com/gallery-so/gallery/assets/80802871/08bc57ef-95a3-4ccc-a9f7-1d76c3bc5f2a



### Edge Cases
Click "ALL GALLERIES" (back button) should continue to publish the gallery and return the user to the multi gallery view.

### Testing Steps
Edit a gallery and confirm the buttons you have to press to save and exit the editor.

### Checklist

Please make sure to review and check all of the following:
